### PR TITLE
 Inference Generator - BatchNorm & Scale Layer Fusion

### PR DESCRIPTION
**Pull request by kiritigowda**
scale_init.f32 file was generated by default, but scale_init.f32 generation is only for unfused batch norm & scale layer.

### CODE REVIEW

**Comment by rgiduthuri** [comment]
The c++11 config in .pro file is already present in the develop branch.
I'm guessing that there is a disconnect between your branch and the develop branch.
Please try to resync and submit another pull request.
